### DISCUSSION TITLE
Can you move Guake to the right side of the screen with `Super + right arrow`?

### DISCUSSION THREAD

**Discussion by paseaf** [Guake/guake#2097] [Q&A] [answered]
Dear Community,

I love the drop-down feature of Guake, but sometimes it would be nice to have the terminal on the right side of the screen.
Is it possible to move Guake to the right side of the screen using `Super + right arrow`?

### Action
`<Super>+<Right Arrow>`

### Current behavior
- Guake goes to the right side at first, but instantly back to the top of the screen.

https://user-images.githubusercontent.com/33207565/173202785-6023e74e-943e-4952-9f52-01674122c839.mp4

### Expected behavior

- Guake goes to the right side, and stays there
Something like
![image](https://user-images.githubusercontent.com/33207565/172118102-5db081eb-7592-46cd-9743-ffcc995c2a10.png)
(It's GNOME Terminal in the screenshot

### My Environment
- OS: Fedora 36
- GNOME shell version: 42.2 
- Guake Terminal version: 3.7.0

Best

**Comment by Davidy22** [comment, marked as answer]
Didn't know about that shortcut previously, but it seems to work trying it out on GNOME shell 42. If you prefer it on the right side of the screen, there are options to choose the alignment and width of the Guake terminal under main window settings.

**Reply by paseaf** [reply]
Thank you for your tips!
For me it also works right after pressing the `Super + right arrow`, but Guake somehow instantly went back to the top afterwards.

I prefer it at top most of the time, only sometimes on the right hand side :)
(Sorry that I wasn't very clear before. Just reformulated my question a bit

**Reply by Davidy22** [reply]
Oh, you mean when you hide and re-show guake it goes back to how it looked before you resized guake? Yeah I see it now, guake isn't capturing those shortcuts so it isn't saving the window size changes for those, we could I guess, ideally I maybe want to hook into a signal sent from GTK telling me that resize has happened.

**Reply by paseaf** [reply]
In my case, after pressing `Super + right arrow`, Guake just stays on the right side for less than 1 second then goes back to top on its own.

Here is the screen recording

<video src="https://user-images.githubusercontent.com/33207565/173202785-6023e74e-943e-4952-9f52-01674122c839.mp4">

But what you said is definitely a nice to have feature :)

**Reply by Davidy22** [reply]
Oh huh, what does `guake --verbose` say when this happens? Not sure if it'll have anything to say about it but might be helpful. Or maybe you have that keybinding bound to a window resize action in Guake preferences? I notice when I move guake to the right then hit a window resize keyboard shortcut it jumps back to the top

**Reply by paseaf** [reply]
Hey David, I just updated Guake to 3.9.0 on Fedora and the problem is gone!
Thank you again for you help!
